Update unit tests data to resolve test failure

Currently the unit test in the pipeline logs the following failure: 

```
=== Failed
=== FAIL: pkg/certifier/certify TestCertify/query_and_generate_attestation (0.51s)
2026-04-11T15:41:12.810Z	INFO	logging/logger.go:79	Logging at debug level	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:12.810Z	DEBUG	logging/logger.go:111	InitLogger has not been called. Defaulting to debug log level	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:12.810Z	INFO	certify/certify.go:109	Starting certifier run: 2026-04-11 15:41:12.810229459 +0000 UTC	{"guac-version": "v0.0.1-custom"}
make: *** [Makefile:21: test] Error 1
2026-04-11T15:41:13.310Z	DEBUG	certify/certify.go:172	starting up the child logger: osv_certifier	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:13.311Z	DEBUG	certify/certify.go:172	starting up the child logger: osv_certifier	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:13.311Z	DEBUG	certify/certify.go:172	starting up the child logger: osv_certifier	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:13.311Z	DEBUG	certify/certify.go:188	starting up the child logger: osv_certifier	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:13.311Z	INFO	certify/certify.go:114	Certifier run completed: 2026-04-11 15:41:13.311415347 +0000 UTC	{"guac-version": "v0.0.1-custom"}
    certify_test.go:220: g.RetrieveArtifacts() = {"type":"https://in-toto.io/Statement/v1","subject":[{"uri":"pkg:maven/org.apache.logging.log4j/log4j-core@2.8.1"}],"predicate_type":"https://in-toto.io/attestation/vulns/v0.2","predicate":{"scanner":{"uri":"osv.dev","version":"0.0.14","db":{},"result":[{"id":"GHSA-3pxv-7cmr-fjr4"},{"id":"GHSA-7rjr-3q55-vv33"},{"id":"GHSA-8489-44mv-ggj8"},{"id":"GHSA-fxph-q3j8-mv87"},{"id":"GHSA-jfh8-c2jp-5v3q"},{"id":"GHSA-p6xc-xr62-6r2g"},{"id":"GHSA-vc5p-v9hr-52mj"},{"id":"GHSA-vwqq-5vrc-xw9h"}]},"metadata":{"scanStartedOn":"2026-04-11T15:41:13.310901081Z","scanFinishedOn":"2026-04-11T15:41:13.310901081Z"}}}, want {
        		"type":"https://in-toto.io/Statement/v1",
        		"subject":[
        		   {
        			  "uri":"pkg:maven/org.apache.logging.log4j/log4j-core@2.8.1"
        		   }
        		],
        		"predicate_type":"https://in-toto.io/attestation/vulns/v0.2",
        		"predicate":{
        		   "scanner":{
        			  "uri":"osv.dev",
        			  "version":"0.0.14",
        			  "db":{
        			  },
        			  "result":[
        				 {
        					"id":"GHSA-7rjr-3q55-vv33"
        				 },
        				 {
        					"id":"GHSA-8489-44mv-ggj8"
        				 },
        				 {
        					"id":"GHSA-fxph-q3j8-mv87"
        				 },
        				 {
        					"id":"GHSA-jfh8-c2jp-5v3q"
        				 },
        				 {
        					"id":"GHSA-p6xc-xr62-6r2g"
        				 },
        				 {
        					"id":"GHSA-vc5p-v9hr-52mj"
        				 },
        				 {
        					"id":"GHSA-vwqq-5vrc-xw9h"
        				 }
        			  ]
        		   },
        		   "metadata":{
        			  "scanStartedOn":"2022-11-22T13:19:18.825699-05:00",
        			  "scanFinishedOn":"2022-11-22T13:19:18.825699-05:00"
        		   }
        		}
        	 }

=== FAIL: pkg/certifier/osv TestOSVCertifier_CertifyVulns (0.60s)
2026-04-11T15:41:15.810Z	INFO	logging/logger.go:79	Logging at debug level	{"guac-version": "v0.0.1-custom"}
2026-04-11T15:41:15.811Z	DEBUG	logging/logger.go:111	InitLogger has not been called. Defaulting to debug log level	{"guac-version": "v0.0.1-custom"}

=== FAIL: pkg/ingestor/parser/common/scanner TestPurlsToScan/valid_vulnerability_certifier_document (0.30s)
    scanner_test.go:374: Unexpected results. (-want +got):
          []assembler.CertifyVulnIngest(Inverse(Sort, []assembler.CertifyVulnIngest{
          	... // 5 identical elements
          	{Pkg: &{Type: "maven", Namespace: &"org.apache.logging.log4j", Name: "log4j-core", Version: &"2.8.1", ...}, Vulnerability: &{Type: "osv", VulnerabilityID: "ghsa-8489-44mv-ggj8"}, VulnData: &{ScannerUri: "osv.dev", ScannerVersion: "0.0.14", Origin: "osv_certifier", Collector: "osv_certifier", ...}},
          	{Pkg: &{Type: "maven", Namespace: &"org.apache.logging.log4j", Name: "log4j-core", Version: &"2.8.1", ...}, Vulnerability: &{Type: "osv", VulnerabilityID: "ghsa-7rjr-3q55-vv33"}, VulnData: &{ScannerUri: "osv.dev", ScannerVersion: "0.0.14", Origin: "osv_certifier", Collector: "osv_certifier", ...}},
        + 	{
        + 		Pkg: &generated.PkgInputSpec{
        + 			Type:      "maven",
        + 			Namespace: &"org.apache.logging.log4j",
        + 			Name:      "log4j-core",
        + 			Version:   &"2.8.1",
        + 			Subpath:   &"",
        + 		},
        + 		Vulnerability: &generated.VulnerabilityInputSpec{Type: "osv", VulnerabilityID: "ghsa-3pxv-7cmr-fjr4"},
        + 		VulnData: &generated.ScanMetadataInput{
        + 			TimeScanned:    s"2026-04-11 15:41:58.511471985 +0000 UTC",
        + 			ScannerUri:     "osv.dev",
        + 			ScannerVersion: "0.0.14",
        + 			Origin:         "osv_certifier",
        + 			Collector:      "osv_certifier",
        + 			DocumentRef:    "sha256_c8f83e4486a184c8da086b7ff45c31e16110be3c9c0ef5c6fc4c8f702"...,
        + 		},
        + 	},
          }))
    scanner_test.go:377: Unexpected results. (-want +got):
          []assembler.VulnEqualIngest(Inverse(Sort, []assembler.VulnEqualIngest{
          	... // 5 identical elements
          	{Vulnerability: &{Type: "osv", VulnerabilityID: "ghsa-8489-44mv-ggj8"}, EqualVulnerability: &{Type: "ghsa", VulnerabilityID: "ghsa-8489-44mv-ggj8"}, VulnEqual: &{Justification: "Decoded OSV data", Origin: "osv_certifier", Collector: "osv_certifier", ...}},
          	{Vulnerability: &{Type: "osv", VulnerabilityID: "ghsa-7rjr-3q55-vv33"}, EqualVulnerability: &{Type: "ghsa", VulnerabilityID: "ghsa-7rjr-3q55-vv33"}, VulnEqual: &{Justification: "Decoded OSV data", Origin: "osv_certifier", Collector: "osv_certifier", ...}},
        + 	{
        + 		Vulnerability:      &generated.VulnerabilityInputSpec{Type: "osv", VulnerabilityID: "ghsa-3pxv-7cmr-fjr4"},
        + 		EqualVulnerability: &generated.VulnerabilityInputSpec{Type: "ghsa", VulnerabilityID: "ghsa-3pxv-7cmr-fjr4"},
        + 		VulnEqual: &generated.VulnEqualInputSpec{
        + 			Justification: "Decoded OSV data",
        + 			Origin:        "osv_certifier",
        + 			Collector:     "osv_certifier",
        + 			DocumentRef:   "sha256_c8f83e4486a184c8da086b7ff45c31e16110be3c9c0ef5c6fc4c8f702"...,
        + 		},
        + 	},
          }))
    --- FAIL: TestPurlsToScan/valid_vulnerability_certifier_document (0.30s)

=== FAIL: pkg/ingestor/parser/common/scanner TestPurlsToScan (18.93s)
```

--- a/internal/testing/testdata/exampledata/certify-vuln-v02.json
+++ b/internal/testing/testdata/exampledata/certify-vuln-v02.json
@@ -11,6 +11,8 @@
       "uri": "osv.dev",
       "version": "0.0.14",
       "result": [{
+            "id": "GHSA-3pxv-7cmr-fjr4"
+        }, {
             "id": "GHSA-7rjr-3q55-vv33",
             "severity": [{
                 "method": "CVSSv31",

--- a/internal/testing/testdata/exampledata/certify-vuln.json
+++ b/internal/testing/testdata/exampledata/certify-vuln.json
@@ -11,6 +11,8 @@
       "uri": "osv.dev",
       "version": "0.0.14",
       "result": [{
+            "id": "GHSA-3pxv-7cmr-fjr4"
+        }, {
             "id": "GHSA-7rjr-3q55-vv33",
             "severity": [{
                 "method": "CVSSv31",

--- a/internal/testing/testdata/testdata.go
+++ b/internal/testing/testdata/testdata.go
@@ -2260,6 +2260,9 @@ var (
 			  },
 			  "result":[
 				 {
+					"id":"GHSA-3pxv-7cmr-fjr4"
+				 },
+				 {
 					"id":"GHSA-7rjr-3q55-vv33"
 				 },
 				 {
@@ -2368,6 +2371,9 @@ var (
 			  "db":{
 			  },
 			  "result":[
+				 {
+					"id":"GHSA-3pxv-7cmr-fjr4"
+				 },
 				 {
 					"id":"GHSA-7rjr-3q55-vv33"
 				 },

--- a/pkg/ingestor/parser/common/scanner/scanner_test.go
+++ b/pkg/ingestor/parser/common/scanner/scanner_test.go
@@ -56,6 +56,27 @@ func TestPurlsToScan(t *testing.T) {
 				},
 				Vulnerability: &generated.VulnerabilityInputSpec{
 					Type:            "osv",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				VulnData: &generated.ScanMetadataInput{
+					TimeScanned:    tm,
+					ScannerUri:     "osv.dev",
+					ScannerVersion: "0.0.14",
+					Origin:         "osv_certifier",
+					Collector:      "osv_certifier",
+					DocumentRef:    "sha256_daeea32fb48a532d48ce7a549b7e0cdf98eb6df80869c3b6d3ec21174b015d14",
+				},
+			},
+			{
+				Pkg: &generated.PkgInputSpec{
+					Type:      "maven",
+					Namespace: ptrfrom.String("org.apache.logging.log4j"),
+					Name:      "log4j-core",
+					Version:   ptrfrom.String("2.8.1"),
+					Subpath:   ptrfrom.String(""),
+				},
+				Vulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "osv",
 					VulnerabilityID: "ghsa-7rjr-3q55-vv33",
 				},
 				VulnData: &generated.ScanMetadataInput{
@@ -195,6 +216,22 @@ func TestPurlsToScan(t *testing.T) {
 			},
 		},
 		wantVEs: []assembler.VulnEqualIngest{
+			{
+				Vulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "osv",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				EqualVulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "ghsa",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				VulnEqual: &generated.VulnEqualInputSpec{
+					Justification: "Decoded OSV data",
+					Origin:        "osv_certifier",
+					Collector:     "osv_certifier",
+					DocumentRef:   "sha256_daeea32fb48a532d48ce7a549b7e0cdf98eb6df80869c3b6d3ec21174b015d14",
+				},
+			},
 			{
 				Vulnerability: &generated.VulnerabilityInputSpec{
 					Type:            "osv",

--- a/pkg/ingestor/parser/vuln/vuln_test.go
+++ b/pkg/ingestor/parser/vuln/vuln_test.go
@@ -101,6 +101,24 @@ func TestParser(t *testing.T) {
 				},
 				Vulnerability: &generated.VulnerabilityInputSpec{
 					Type:            "osv",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				VulnData: &generated.ScanMetadataInput{
+					TimeScanned:    tm,
+					ScannerUri:     "osv.dev",
+					ScannerVersion: "0.0.14",
+				},
+			},
+			{
+				Pkg: &generated.PkgInputSpec{
+					Type:      "maven",
+					Namespace: ptrfrom.String("org.apache.logging.log4j"),
+					Name:      "log4j-core",
+					Version:   ptrfrom.String("2.8.1"),
+					Subpath:   ptrfrom.String(""),
+				},
+				Vulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "osv",
 					VulnerabilityID: "ghsa-fxph-q3j8-mv87",
 				},
 				VulnData: &generated.ScanMetadataInput{
@@ -217,6 +235,19 @@ func TestParser(t *testing.T) {
 				EqualVulnerability: &generated.VulnerabilityInputSpec{
 					Type:            "ghsa",
 					VulnerabilityID: "ghsa-8489-44mv-ggj8",
+				},
+				VulnEqual: &generated.VulnEqualInputSpec{
+					Justification: "Decoded OSV data",
+				},
+			},
+			{
+				Vulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "osv",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				EqualVulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "ghsa",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
 				},
 				VulnEqual: &generated.VulnEqualInputSpec{
 					Justification: "Decoded OSV data",
@@ -374,6 +405,24 @@ func TestParser(t *testing.T) {
 				},
 				Vulnerability: &generated.VulnerabilityInputSpec{
 					Type:            "osv",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				VulnData: &generated.ScanMetadataInput{
+					TimeScanned:    tm,
+					ScannerUri:     "osv.dev",
+					ScannerVersion: "0.0.14",
+				},
+			},
+			{
+				Pkg: &generated.PkgInputSpec{
+					Type:      "maven",
+					Namespace: ptrfrom.String("org.apache.logging.log4j"),
+					Name:      "log4j-core",
+					Version:   ptrfrom.String("2.8.1"),
+					Subpath:   ptrfrom.String(""),
+				},
+				Vulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "osv",
 					VulnerabilityID: "ghsa-fxph-q3j8-mv87",
 				},
 				VulnData: &generated.ScanMetadataInput{
@@ -490,6 +539,19 @@ func TestParser(t *testing.T) {
 				EqualVulnerability: &generated.VulnerabilityInputSpec{
 					Type:            "ghsa",
 					VulnerabilityID: "ghsa-8489-44mv-ggj8",
+				},
+				VulnEqual: &generated.VulnEqualInputSpec{
+					Justification: "Decoded OSV data",
+				},
+			},
+			{
+				Vulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "osv",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
+				},
+				EqualVulnerability: &generated.VulnerabilityInputSpec{
+					Type:            "ghsa",
+					VulnerabilityID: "ghsa-3pxv-7cmr-fjr4",
 				},
 				VulnEqual: &generated.VulnEqualInputSpec{
 					Justification: "Decoded OSV data",

